example_tracepoint_test.go

```diff
@@ -0,0 +1,110 @@
+// +build linux
+
+package ebpf_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	ringbuffer "github.com/cilium/ebpf/perf"
+
+	"github.com/elastic/go-perf"
+)
+
+// This demonstrates how to attach an eBPF program to a tracepoint.
+// The program will be attached to the sys_enter_open syscall and print out the integer
+// 123 everytime the sycall is used.
+func main() {
```

Comment:
Doesn't this have to be something like `func Example_quuxBar`?

Reply:
I was testing this program in a different context before :grin: and forgot to rename it.